frontend/containers/forms/filters/component.tsx

```diff
@@ -110,80 +116,148 @@ export const Filters: FC<FiltersProps> = ({
-            <label htmlFor="verified" className="mt-1 text-sm font-normal text-gray-800">
-              <FormattedMessage defaultMessage="Show verified content only" id="i9eK6b" />
-            </label>
-          </div>
-        </div>
-  */}
+      {!filters ? (
+        <Loading />
+      ) : (
+        <form className="relative flex flex-col p-6" onSubmit={handleSubmit(onSubmitFilters)}>
+          <Button
+            theme="naked"
+            size="smallest"
+            className="absolute top-6 right-6"
+            onClick={closeFilters}
+          >
+            <span className="sr-only">
+              <FormattedMessage defaultMessage="Close filters" id="78vrMq" />
+            </span>
+            <CloseIcon className="w-4 h-4 transition-transform rotate-0 hover:rotate-180" />
+          </Button>
+          <div className="overflow-y-auto">
+            {/* VERIFICATION FILTERS: HIDDEN
+            <div className="flex justify-between">
+              <div className="flex items-center mb-4">
+                <Checkbox
+                  id="verified"
+                  name="only_verified"
+                  register={register}
+                  registerOptions={{ onChange: onCheckboxChange }}
+                />
+                <label htmlFor="verified" className="mt-1 text-sm font-normal text-gray-800">
+                  <FormattedMessage defaultMessage="Show verified content only" id="i9eK6b" />
+                </label>
+              </div>
+            </div> */}
+            <div className="flex flex-col flex-wrap gap-y-4">
+              {filterTexts?.map((filter) => {
+                return (
+                  <div key={filter.title}>
+                    <fieldset>
+                      <legend className="inline mb-3 font-sans text-base font-medium text-black">
+                        <span className="mr-2 font-sans text-sm font-semibold text-gray-800">
+                          {filter.title}
+                        </span>
+                        <FieldInfo
+                          content={
+                            filter.description ? (
+                              filter.description
+                            ) : (
+                              <ul>
+                                {filter.values?.map((value) => (
+                                  <li key={value.id} className="mb-2">
+                                    <p className="font-bold ">{value.name}</p>
+                                    {!!value.description && <p>{value.description}</p>}
+                                  </li>
+                                ))}
+                              </ul>
+                            )
+                          }
+                        />
+                      </legend>
 
-        <div className="flex flex-col flex-wrap gap-y-4">
-          {filters?.map((filter) => {
-            return (
-              <div key={filter.title}>
+                      {!!filter.values?.length && (
```

Comment:
Could you add a small padding so that the focus ring is fully visible?
![The focus ring is cut on the left](https://user-images.githubusercontent.com/6073968/190175339-f39d9c9d-0477-434a-86bc-7e899eeda188.png)